### PR TITLE
👌IMPROVE: PHPCS Test

### PR DESCRIPTION
**Pull request by ahmadbilaldev**

### CODE REVIEW

**Comment by ahmadbilaldev** [comment]
GitHub API communication error. Please contact a human. (commit-ID: 2ab3e860159a5cb2d0d8f4d729f3dd320ac85c1c).

File: prism-languages.php [hunk 1]
```diff
@@ -63,4 +62,5 @@ function mkaz_code_syntax_block_get_supported_languages() {
 	 * @param string $languages Array of languages supported
 	 */
 	return apply_filters( 'mkaz_code_syntax_language_list', $languages );
+	// Test PHPCS check, should give warning for this comment
```
Comment (theprismane) [inline prism-languages.php:65]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

File: prism-languages.php [hunk 2]
```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
 		"typescript" => "TypeScript",
 		"vim" => "vim",
-		"wasm" => "WebAssembly",
 		"wiki" => "Wiki markup",
 		"xml" => "XML",
 		"yaml" => "YAML",
```
Comment (theprismane) [inline prism-languages.php:50]: :no_entry_sign: **Error**: String &quot;YAML&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

Comment (theprismane) [inline prism-languages.php:50]: :warning: **Warning**: Array double arrow not aligned correctly; expected 7 space(s) between &quot;&quot;yaml&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (theprismane) [inline prism-languages.php:50]: :no_entry_sign: **Error**: String &quot;yaml&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

**Review by theprismane — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 17 errors 

:warning: 6 warnings 

***

**hashes-api**-scanning skipped


Posting will continue in further review(s)

File: prism-languages.php [hunk 3]
```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
 		"typescript" => "TypeScript",
 		"vim" => "vim",
-		"wasm" => "WebAssembly",
 		"wiki" => "Wiki markup",
 		"xml" => "XML",
```
Comment (theprismane) [inline prism-languages.php:49]: :no_entry_sign: **Error**: String &quot;XML&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

Comment (theprismane) [inline prism-languages.php:49]: :warning: **Warning**: Array double arrow not aligned correctly; expected 8 space(s) between &quot;&quot;xml&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (theprismane) [inline prism-languages.php:49]: :no_entry_sign: **Error**: String &quot;xml&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

File: prism-languages.php [hunk 4]
```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
 		"typescript" => "TypeScript",
 		"vim" => "vim",
-		"wasm" => "WebAssembly",
 		"wiki" => "Wiki markup",
```
Comment (theprismane) [inline prism-languages.php:48]: :no_entry_sign: **Error**: String &quot;Wiki markup&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

Comment (theprismane) [inline prism-languages.php:48]: :warning: **Warning**: Array double arrow not aligned correctly; expected 7 space(s) between &quot;&quot;wiki&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (theprismane) [inline prism-languages.php:48]: :no_entry_sign: **Error**: String &quot;wiki&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

File: prism-languages.php [hunk 5]
```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
 		"typescript" => "TypeScript",
 		"vim" => "vim",
```
Comment (theprismane) [inline prism-languages.php:47]: :warning: **Warning**: Array double arrow not aligned correctly; expected 8 space(s) between &quot;&quot;vim&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (theprismane) [inline prism-languages.php:47]: :no_entry_sign: **Error**: String &quot;vim&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

File: prism-languages.php [hunk 6]
```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
 		"typescript" => "TypeScript",
```
Comment (theprismane) [inline prism-languages.php:46]: :no_entry_sign: **Error**: String &quot;TypeScript&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

Comment (theprismane) [inline prism-languages.php:46]: :no_entry_sign: **Error**: String &quot;typescript&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

File: prism-languages.php [hunk 7]
```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
```
Comment (theprismane) [inline prism-languages.php:45]: :no_entry_sign: **Error**: String &quot;TOML&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

Comment (theprismane) [inline prism-languages.php:45]: :warning: **Warning**: Array double arrow not aligned correctly; expected 7 space(s) between &quot;&quot;toml&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

Comment (theprismane) [inline prism-languages.php:45]: :no_entry_sign: **Error**: String &quot;toml&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

File: prism-languages.php [hunk 8]
```diff
@@ -5,7 +5,7 @@
  * @return array Hash of lang => language
  */
 function mkaz_code_syntax_block_get_supported_languages() {
-	
+
 	$languages = array(
 		"apacheconf" => "Apache Config",
 		"bash" => "Bash/Shell",
```
Comment (theprismane) [inline prism-languages.php:11]: :no_entry_sign: **Error**: String &quot;Bash/Shell&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

Comment (theprismane) [inline prism-languages.php:11]: :warning: **Warning**: Array double arrow not aligned correctly; expected 7 space(s) between &quot;&quot;bash&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).

**Review by theprismane — CHANGES_REQUESTED** [review]
Previous scan continued.

File: prism-languages.php (hunk 8 above)
Comment (theprismane) [inline prism-languages.php:11]: :no_entry_sign: **Error**: String &quot;bash&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

**Review by theprismane — CHANGES_REQUESTED** [review]
Previous scan continued.

File: prism-languages.php [hunk 9]
```diff
@@ -5,7 +5,7 @@
  * @return array Hash of lang => language
  */
 function mkaz_code_syntax_block_get_supported_languages() {
-	
+
 	$languages = array(
 		"apacheconf" => "Apache Config",
```
Comment (theprismane) [inline prism-languages.php:10]: :no_entry_sign: **Error**: String &quot;Apache Config&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

Comment (theprismane) [inline prism-languages.php:10]: :no_entry_sign: **Error**: String &quot;apacheconf&quot; does not require double quotes; use single quotes instead (*Squiz.Strings.DoubleQuoteUsage.NotRequired*).

File: prism-languages.php [hunk 10]
```diff
@@ -5,7 +5,7 @@
  * @return array Hash of lang => language
```
Comment (theprismane) [inline prism-languages.php:5]: :no_entry_sign: **Error**: There must be exactly one blank line before the tags in a doc comment (*Generic.Commenting.DocComment.SpacingBeforeTags*).

**Pull request closed without merging**